### PR TITLE
Feature/mw 21

### PR DESCRIPTION
**Pull request by Aaliyah379**
Please review the code.
Thanks

### CODE REVIEW

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 1]
```diff
@@ -90,7 +88,7 @@ await _unitOfWork.ExerciseRepository
 
             await _unitOfWork.Save();
 
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction(nameof(ExercisesController.Details), "WorkoutDays", new { ID = exercise.WorkoutDayId });
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: new { Id = exercise.WorkoutDayId } according to camel Case rule

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 2]
```diff
@@ -147,7 +145,7 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,Name,RestTime,WorkoutDay
 
                 throw;
             }
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction(nameof(ExercisesController.Details), "WorkoutDays", new { ID = exercise.WorkoutDayId });
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: the same Id instead of ID

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 3]
```diff
@@ -172,12 +171,12 @@ public async Task<IActionResult> Delete(int? id)
         // POST: Exercises/Delete/5
         [HttpPost, ActionName("Delete")]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DeleteConfirmed(int id)
+        public async Task<IActionResult> DeleteConfirmed(int id, int workoutdayId)
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: workoutdayId -> workoutDayId

File: MyWorkout.Web/Controllers/RepeatsController.cs [hunk 4]
```diff
@@ -89,7 +89,7 @@ public async Task<IActionResult> Create([Bind("Id,Number,Count,ExerciseId")] Rep
                 await _unitOfWork.RepeatRepository.Create(repeat);
                 await _unitOfWork.Save();
 
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction(nameof(ExercisesController.Details), "Exercises", new {ID =repeat.ExerciseId});
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/RepeatsController.cs]: ID -> Id

File: MyWorkout.Web/Controllers/RepeatsController.cs [hunk 5]
```diff
@@ -145,7 +145,7 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,Number,Count,ExerciseId"
 
                     throw;
                 }
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction(nameof(ExercisesController.Details),"Exercises", new {ID =repeat.ExerciseId});
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/RepeatsController.cs]: ID -> Id

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs [hunk 6]
```diff
@@ -85,7 +85,7 @@ public async Task<IActionResult> Create([Bind("Id,DayOfWeek,MuscleGroup,PlanId")
                 await _unitOfWork.WorkoutDayRepository.Create(workoutDay);
                 await _unitOfWork.Save();
 
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction(nameof(PlansController.Details), "Plans", new {ID = workoutDay.PlanId });
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/WorkoutDaysController.cs]: ID -> Id

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs [hunk 7]
```diff
@@ -136,7 +136,7 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,DayOfWeek,MuscleGroup, P
 
                     throw;
                 }
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction(nameof(PlansController.Details), "Plans", new {ID = workoutDay.PlanId });
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/WorkoutDaysController.cs]: ID -> Id

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs [hunk 8]
```diff
@@ -163,17 +163,24 @@ public async Task<IActionResult> Delete(int? id)
...
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction(nameof(PlansController.Details), "Plans", new { Id = planId });
         }
 
         private Task<bool> WorkoutDayExists(int id)
         {
             return _unitOfWork.WorkoutDayRepository.IsExist(e => e.Id == id);
         }
     }
+
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/WorkoutDaysController.cs]: remove please white spaces

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs [hunk 9]
```diff
@@ -163,17 +163,24 @@ public async Task<IActionResult> Delete(int? id)
...
 }
+
+        
+    
+
+
+
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/WorkoutDaysController.cs]: and here

Comment (YuriyMorozyuk95) [reply MyWorkout.Web/Controllers/WorkoutDaysController.cs]: white spaces

File: MyWorkout.Web/Controllers/ExercisesController.cs (hunk 1 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/ExercisesController.cs]: Updated

File: MyWorkout.Web/Controllers/ExercisesController.cs (hunk 2 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/ExercisesController.cs]: Updated

File: MyWorkout.Web/Controllers/ExercisesController.cs (hunk 3 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/ExercisesController.cs]: Updated

File: MyWorkout.Web/Controllers/RepeatsController.cs (hunk 4 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/RepeatsController.cs]: Updated

File: MyWorkout.Web/Controllers/RepeatsController.cs (hunk 5 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/RepeatsController.cs]: Updated

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs (hunk 7 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/WorkoutDaysController.cs]: Updated

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 10]
```diff
@@ -56,7 +54,7 @@ public async Task<IActionResult> Details(int? id)
 
             var exercise = await _unitOfWork.ExerciseRepository
                 .Read(id.Value);
-            _unitOfWork.Load(exercise, e => (IEnumerable<Repeat>)e.Repeats);
+            _unitOfWork.Load(exercise, e => (IEnumerable<Exercise>)e.Repeats);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: Why IEnumerable(Excercise) should be IEnumerable(Repeat)

Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/ExercisesController.cs]: I have changed twice maybe, but the error was still there, so I probably forgot to change it back .. Sorry :-(

**Pull request merged**